content/client-lib-development-guide/features.textile

```diff
@@ -558,6 +558,7 @@ h3(#realtime-presence). Presence
 * @(RTP11)@ @Presence#get@ function:
 ** @(RTP11a)@ Returns the list of current members on the channel in a callback. By default, will wait for the @SYNC@ to be completed, see "RTP11c1":#RTP11c1
 ** @(RTP11b)@ Implicitly attaches the @Channel@ if the channel is in the @INITIALIZED@ state. However, if the channel is in or enters the @DETACHED@ or @FAILED@ state before the operation succeeds, it will result in an error
+** @(RTP11d)@ If the @Channel@ is in the @SUSPENDED@ state then the @get@ function immediately returns the members stored in the @PresenceMap@ and does not wait until the channel state changes to @ATTACHED@ irrespective of whether @waitForSync@ is true or false
```

Comment:
It was synchronised previously, but is known to be getting stale.

In the `disconnected` or `connecting` state, you get the current (potentially slightly out-of-date) state, but you know that you will also see one or other of the following events within a known and bounded amount of time:

- a `connected` event, after which you know you will see any events for updates that were not in the state you saw when you called `get()`;

- a `suspended` event, which you know means that you no longer have any assurance of synchronisation, and no timescale that you can rely on for any future change in that status.

So the difference I think is that in `suspended` all you know is that it is potentially stale to an indefinite degree. In the other states, you know that within a bounded time you will either see any events necessary to bring it up to date, or an event telling you that it's in the indefinitely stale state.